lib/cred_stash.rb

```diff
@@ -16,9 +16,9 @@ def get(name, context: {}, raise_if_missing: false)
       nil
     end
 
-    def put(name, value, context: {})
+    def put(name, value, kms_key_id: nil, context: {})
```

Comment:
If `put` is called without `kms_key_id`, it seems to pass `nil` as `key_id` to `generate_data_key`. 
I think that the default value should be specified at here.

```ruby
def put(name, value, kms_key_id: 'alias/credstash', context: {})
```